tests/integration/conftest.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for Jenkins-k8s-operator charm integration tests."""
+
+import pytest
+import pytest_asyncio
+from juju.application import Application
+from juju.client._definitions import FullStatus
+from juju.model import Model
+from pytest import FixtureRequest
+from pytest_operator.plugin import OpsTest
+
+
+@pytest.fixture(scope="module", name="model")
+def model_fixture(ops_test: OpsTest) -> Model:
+    """The testing model."""
+    assert ops_test.model
+    return ops_test.model
+
+
+@pytest.fixture(scope="module", name="jenkins_image")
+def jenkins_image_fixture(request: FixtureRequest) -> str:
+    """The OCI image for Jenkins charm."""
+    jenkins_image = request.config.getoption("--jenkins-image")
+    assert jenkins_image, "--jenkins-image argument was not provided."
```

Comment:
Do you think this should tell the developer that the image is required?

Reply:
At the moment it is saying "not provided" compared to a nice user message such as "the --jenkins-image argument is required which should contain the name of the OCI image" or something like that?

Reply:
Updated thanks!